File: generated-sources/dart-jaguar/mojang-authentication/lib/model/username_password.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'username_password.jser.dart';

class UsernamePassword {
   /* The Mojang account e-mail or username. Never store it. */
  @Alias('username')
  final String username;
   /* The Mojang account password, never store it. */
  @Alias('password')
  final String password;
  

  UsernamePassword(
    

{
    
     this.username = null,  
     this.password = null 
    }
  );

  @override
  String toString() {
    return 'UsernamePassword[username=$username, password=$password, ]';
  }
}

@GenSerializer()
class UsernamePasswordSerializer extends Serializer<UsernamePassword> with _$UsernamePasswordSerializer {

}

File: generated-sources/php-ze-ph/mojang-authentication/src/App/DTO/UsernamePassword.php
<?php

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class UsernamePassword
{
    /**
     * The Mojang account e-mail or username. Never store it.
     * @DTA\Data(field="username")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $username;
    /**
     * The Mojang account password, never store it.
     * @DTA\Data(field="password")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $password;
}
